cartridge-cli.lua

```diff
@@ -9,9 +9,6 @@ local log = require('log')
 local socket = require('socket')
 local yaml = require('yaml')
 
-local luarocks_searcher = require('luarocks.search')
```

Comment:
Make sure that we don't use luarocks 3 in another parts of cartridge-cli

Reply:
`luarocks.*` modules was used only for version detection, it you mean that.